src/Client.ts

```diff
@@ -273,7 +273,7 @@ export class Client {
         profiles: [profile],
         ephemeral: false,
         config,
-        source: {
+        source: typeof image === !== 'string' ? image : {
```

Comment:
I'm not sure of the comparing voodoo that you have here :joy:. We also use double quotes for strings.

I believe it can be simplified to:
``` suggestion
        source: typeof image !== "string" ? image : {
```

Agree? Or am I missing something?